src/components/StoryFooter/Payout.less

```diff
@@ -0,0 +1,18 @@
+.Payout {
+  position: absolute;
+  right: 0;
+  color: #99aab5;
```

Comment:
colors can probably go in `styles/custom.less`.

I usually name colors by their `hue-colorName`, I get the color name from here: (http://www.color-blindness.com/color-name-hue/)

So for `#99aab5` it would be, `@blue-botticelli: #99aab5`